blueprints/hr_assistant/domains/date/get_date/train.txt

```diff
@@ -0,0 +1,203 @@
+{julia|name} {birthday|dob}
+tell me {birth date|dob} of {Ermine Pelletier|name}
+What was the exact date when {desiree|name} was {hired|employment_action} at our company?
+{start date|employment_action}, {Bianca Roehrich|name}
+when did {Amy|name} officially {join|employment_action} the company
+find me the official {start date|employment_action} of {Nan|name} at this company
+What was it that {Nilson Ferivandes|name} {began working for|employment_action} this company?
+{Quinn Rarrick|name} {bday month|dob}
+{ivan|name} {start date|employment_action}
+When was {Lenora Tejeda|name} {born|dob}?
+{Ivan Rogers|name} {bday month|dob}
+When {Trina Alagbe|name} was {born|dob}, what was the month
+How long ago did we {fire|employment_action} {nan|name} from the company
+How long has Mr.{lin chan|name} been with the company?
+Can you tell me how long has {Carlos|name} worked in the firm
+whats the date of {joining|employment_action} for ms.{Morway|name}
+When was {Julia|name} {fired|employment_action}?
+{Leigh Smith|name} {date of birth|dob}
+What was the exact date when {Amy|name} {started her job|employment_action} at our company?
+What was {Brandon LeBlanc|name}s date when he started to {work here|employment_action}?
```

Comment:
{Brandon LeBlanc|name}'s (weird sentence. What was the date when...?)
Also, shouldn't the `employment_action` entity be `started to work here`?